crypto/x509/t_acert.c

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright 2021 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <stdio.h>
+#include "internal/cryptlib.h"
+#include <openssl/buffer.h>
+#include <openssl/bn.h>
+#include <openssl/objects.h>
+#include <openssl/x509.h>
+#include <openssl/x509v3.h>
+
+int X509_ACERT_print_ex(BIO *bp, X509_ACERT *x, unsigned long nmflags,
+                      unsigned long cflag)
+{
+    long l;
+    int i;
+    const STACK_OF(X509_EXTENSION) *exts;
+    char mlch = ' ';
+
+    if ((nmflags & XN_FLAG_SEP_MASK) == XN_FLAG_SEP_MULTILINE) {
+        mlch = '\n';
+    }
+
+    if (!(cflag & X509_FLAG_NO_HEADER)) {
```

Comment:
nit : `    if ((cflag & X509_FLAG_NO_HEADER) != 0) {`
And so on...

Reply:
`if ((cflag & X509_FLAG_NO_HEADER) == 0) {`  I think.
The checks in t_x509.c, and t_req.c all use the `if (!(cflag & XXX))`  format, so
I thought it was better to keep consistent with the existing code (albeit in a different file).

Reply:
The Coding GuideStyle is to explicitly test this binary bit-fields:
`if ((cflag & X509_FLAG_NO_HEADER) == 0) {` , you're right

Reply:
It does?  I'm looking at the style guide at https://www.openssl.org/policies/codingstyle.html and the closest I can find
is Chapter 21, which says that `if (! (...) ) ` shouldn't be used for pointer checks and str...cmp functions.  
I can't seem to find anything about binary bit fields.  Am I looking at the wrong guide? or just the wrong place in the right guide? :)

Reply:
See #10725 , which try to grab and extend the current codingstyle.

Reply:
Great, thanks.  This is really helpful.  I'll look it over the PR again with these points in mind.